File: 1480. Running Sum of 1D Array/Solution.cs
public class Solution {
    public int[] RunningSum(int[] nums) {
        int current = 0;
        for(int i = 0; i < nums.Length; i++){
            current += nums[i];
            nums[i] = current;
        }
        return nums;
    }
}

// complexity
// O(n) Time Complexity
// O(1) Space Complexity

File: 1480. Running Sum of 1D Array/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        int current = 0;
        for(int i = 0; i < nums.size(); i++){
            current += nums[i];
            nums[i] = current;
        }
        return nums;
    }
};

// Time: O(n)
// Space: O(1)